### PR TITLE
docs: fix typo

### PR DESCRIPTION
**Pull request by chenbin92**
##### Checklist

- [x] `npm test` passes
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change

fix typo：

- wil  -> will
- `String|Array`  ->  `String/Array` 
- No `Application` method in `egg-core`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -226,8 +226,8 @@ Param          | Type           | Description
 -------------- | -------------- | ------------------------
 directory      | `String/Array` | directories to be loaded
 target         | `Object`       | attach the target object from loaded files
-match          | `String|Array` | match the files when load, default to `**/*.js`
-ignore         | `String|Array` | ignore the files when load
+match          | `String/Array` | match the files when load, default to `**/*.js`
+ignore         | `String/Array` | ignore the files when load
```
Comment (popomore) [inline README.md]: 这里会有问题？

File: README.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ Directory structure
 Then you can start with code below
 
 ```js
-const Application = require('egg-core').Application;
+const Application = require('egg-core');
```
Comment (popomore) [inline README.md]: 应该是 EggCore

File: README.md (hunk 1 above)
Comment (chenbin92) [inline README.md]: 如下图：MarkDown 语法问题，写成 `|` 会忽略后面的注释

screen

File: README.md (hunk 2 above)
Comment (chenbin92) [inline README.md]: 更新了

**Pull request merged**